CriticalMapsSnapshotTests/MockThemeController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SnapshotTester.swift
+//  CriticalMapsSnapshotTests
+//
+//  Created by MAXIM TSVETKOV on 05.10.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+@testable import CriticalMaps
+import SnapshotTesting
+import XCTest
+
+class MockThemeController: ThemeController {
+    static let shared = MockThemeController()
+    
+    private init() {
+        let store = MockThemeStore()
+        super.init(store: store)
+    }
+}
+
+//TODO: move MockThemeStore class to shared file for both CriticalMassTests and CriticalMapsSnapshotTests targets
```

Comment:
Are you planing to resolve that TODO before we merge it to master. We should create an issue if we need to follow up

Reply:
Resolved by sharing some mock classes between test targets